kokoro/builds/build.sh

```diff
@@ -15,7 +15,7 @@ if [ -z "${CONAN_PROFILE}" ]; then
   declare -A profile_mapping=( \
     [ggp_relwithdebinfo]=skip \
     [msvc2019_relwithdebinfo]=skip \
-    [clang7_relwithdebinfo]=skip \
+    [clang7_relwithdebinfo]=clang11_release_system_deps \
```

Comment:
Reusing the clang7 job is definitely a hack, but the best thing we can do right now without changing the job config.

Reply:
Maybe you can add a comment to the file?